auxprogs/joingenes/Makefile

```
@@ -1,20 +1,20 @@
 CC=g++
```

```suggestion
CXX?=g++
```
in this file macro CC is defined as g++ -> better use CXX and make it overidable (as in auxprogs/homGeneMapping and /Makefile src/Makefile)